developer-docs-site/docs/guides/getting-started.md

```diff
@@ -12,7 +12,7 @@ To kick-start your journey as a developer in the Aptos ecosystem, set up your de
 
 ## Clone the Aptos-core repo
 
-Start by cloning the `aptos-core` GitHub repo from [[GitHub](https://github.com/aptos-labs/aptos-core)](https://github.com/aptos-labs/aptos-core).
+Start by cloning the `aptos-core` GitHub repo from [GitHub](https://github.com/aptos-labs/aptos-core).
```

Comment:
I thought I fixed all these to add a `.git` extension, apparently not. You can add or I will clean it up later.